webrtc.html

```diff
@@ -1313,8 +1308,55 @@
               </li>
 
               <li>
-                <p>If the candidate is successfully applied, resolve
-                <var>p</var> with <code>undefined</code>.</p>
+                <p>In parallel, start the process to apply
+                <var>candidate</var>.</p>
+
+                <div class="note">We probably want to define the process to
+                'apply an ICE candidate' in JSEP and link to it above.</div>
+
+                <ol>
+                  <li>
+                    <p>If the candidate could not be successfully applied,
+                    reject <var>p</var> with a <code>DOMException</code> object
+                    whose <code>name</code> attribute has the value
+                    <code>OperationError</code> and abort these steps.</p>
+                  </li>
+
+                  <li>
+                    <p>If <var>description</var> is applied successfully, the
+                    User Agent MUST queue a task that runs the following
+                    steps:</p>
+
+                    <ol>
+                      <li>
+                        <p>Let <var>remoteDescription</var> be
+                        <var>connection</var>'s <code><a href=
+                        "#dom-peerconnection-pendingremotedesc">pendingRemoteDescription</a>
+                        </code> if not null, otherwise <var>connection</var>'s
+                        <code><a href=
+                        "#dom-peerconnection-currentremotedesc">currentRemoteDescription</a></code>.</p>
+                      </li>
+
+                      <li>
+                        <p>Add <var>candidate</var> to
+                        <var>remoteDescription</var>.</p>
+                      </li>
+
+                      <li>
+                        <p>If the <a>ICE Agent</a> is not currently checking
+                        candidate pairs, the <a>ICE Agent</a> MUST start
+                        checking candidate pairs and <a data-lt=
+                        "update the ICE connection state">update
+                        <var>connection</var>'s ICE connection state</a> to
+                        <code>checking</code>.</p>
```

Comment:
Where's the transition state diagram for the ICE connection state?
Currently (editors' draft) I find two references to "update the ICE connection state", one to "connected" and the other one to either of "completed" or "failed". there is no transition to "closed" or "disconnected" that I can find.
If it can only transition to "closed" together with the RTCSignalingState, we're OK, but then *that* needs to be documented.